crates/sui-indexer-alt-framework/src/ingestion/broadcaster.rs

```diff
@@ -138,11 +129,11 @@ where
     })
 }
 
-/// Fetch and broadcasts checkpoints from a range to subscribers.
+/// Fetch and broadcasts checkpoints from a range [start..end) to subscribers.
 /// This task is ingest_hi-aware and will wait if it encounters a checkpoint
 /// beyond the current ingest_hi, resuming when ingest_hi advances to currently
 /// ingesting checkpoints.
-async fn broadcast_range(
+async fn ingest_and_broadcast_range(
```

Comment:
Name change here to distinguish from `streaming_and_broadcast_range` coming later.